### PR TITLE
Formatting update

### PR DESCRIPTION
**Pull request by ymmit85**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,14 +98,14 @@ The **Healthcheck** sub-schema is used to toggle health checks on or off. Curren
 
 ## Examples 
 - Generate HTML & Word reports with Timestamp
-Generate a vROps As Built Report for vROps Server `vrops-01.corp.local` using specified credentials. Export report to HTML & DOC formats. Use default report style. Append timestamp to report filename. Save reports to `C:\Users\Tim\Documents`
+Generate a vROps As Built Report for vROps Server `vrops-01.corp.local` using specified credentials. Export report to HTML & DOC formats. Use default report style. Append timestamp to report filename. Save reports to `C:\Users\User\Documents`
```
Comment (tpcarman) [inline README.md:101]: Update to DOCX format; `Export report to HTML & DOCX formats.`

File: README.md [hunk 2]
```diff
@@ -69,17 +69,18 @@ The **InfoLevel** sub-schema allows configuration of each section of the report
...
+| InfoLevel | Users | 0
+| InfoLevel | Adapters | 1
+| InfoLevel | RemoteCollectors | 1
+| InfoLevel | Alerts | 2
+| InfoLevel | SuperMetrics | 1
+| InfoLevel | ServiceStatus | 1
+| InfoLevel | CustomGroups | 1
+| InfoLevel | Reports | 1
 
 There are 4 levels (0-3) of detail granularity for each section as follows;
```
Comment (tpcarman) [inline README.md:86]: The InfoLevel naming does not align to the report JSON

File: AsBuiltReport.VMware.vROps.json [hunk 3]
```diff
@@ -6,22 +6,23 @@
...
-        "Groups": 3,
-        "Users": 3,
-        "Adapters": 3,
-        "RemoteCollectors": 3,
-        "Alerts": 3,
-        "Symptoms": 3,
-        "SuperMetrics": 3,
-        "ServiceStatus": 3,
-        "CustomGroups": 3
+        "_comment_": "0 = Disabled, 1 = Informative, 2 = Comprehensive",
```
Comment (tpcarman) [inline AsBuiltReport.VMware.vROps.json:14]: InfoLevel naming does not align to README

**Review by tpcarman — CHANGES_REQUESTED** [review]
Review comments inline. Also, please update sample reports showing use of correct font.

File: AsBuiltReport.VMware.vROps.json (hunk 3 above)
Comment (ymmit85) [reply AsBuiltReport.VMware.vROps.json:14]: Updated README

File: README.md (hunk 2 above)
Comment (ymmit85) [reply README.md:86]: Updated README

**Review by tpcarman — APPROVED** [review]

**Pull request merged**